### PR TITLE
Add styles for activity comments

### PR DESCRIPTION
**Pull request by zzmedia**
- added styles for activity comments to the styleguide

https://www.openproject.org/work_packages/13954

### CODE REVIEW

File: app/assets/stylesheets/content/_components_add_comments_onclick.sass [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * See doc/COPYRIGHT.rdoc for more details.  ++*/
+
```
Comment (myabc) [inline app/assets/stylesheets/content/_components_add_comments_onclick.sass:27]: @zzmedia there aren't any styles in this file currently. do we still need it?

Comment (zzmedia) [inline app/assets/stylesheets/content/_components_add_comments_onclick.sass:27]: @myabc I just added it for future purposes, if it's wrong feel free to delete it :-)

Comment (zzmedia) [inline app/assets/stylesheets/content/_components_add_comments_onclick.sass:27]: @myabc do we need the copyright in every sass file? I was not sure about that.

Comment (myabc) [inline app/assets/stylesheets/content/_components_add_comments_onclick.sass:27]: @zzmedia 

>  do we need the copyright in every sass file? I was not sure about that.

I believe so, but "IANAL" - perhaps @ulferts can clarify.

**Pull request merged**